routes/v1/main.route.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const { mainController } = require("../../controllers");
+
+const router = express.Router();
+
+router.route("/").get(mainController.getMain).post(mainController.postMain);
+router.route("/product").post(mainController.createProduct).get(mainController.getProduct);
```

Comment:
Codacy has a fix for the issue: [Replace `.route("/product").post(mainController.createProduct)` with `⏎··.route("/product")⏎··.post(mainController.createProduct)⏎··`](https://app.codacy.com/gh/ParmenidesSartre/Respond-Task1/pullRequest?prid=9934263)
```suggestion
router
```